skills/odl-pdf/scripts/sync-skill-refs.py

```
@@ -0,0 +1,195 @@
+    new_options = sorted(json_names - matrix_names)      # in JSON but not in matrix
+    removed_options = sorted(matrix_names - json_names)  # in matrix but not in JSON
+
+    drift_detected = bool(new_options or removed_options)
+
+    if not drift_detected:
+        print("\u2713 No drift detected.")
+        return 0
+
+    # Report drift
+    if new_options:
+        print(f"\nNEW options (in options.json, not in skill):")
```

Ruff (0.15.9)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)